frontend/src/utils/index.js

```diff
@@ -539,6 +539,19 @@ export function maintenanceWindowWithBeginAndTimezone (beginTime, beginTimezone,
   return { begin, end }
 }
 
+export function maintenanceWindowDuration (windowBegin, windowEnd) {
+  const beginMoment = moment.utc(windowBegin, 'HH:mm')
+  let endMoment = moment.utc(windowEnd, 'HH:mm')
+  if (!beginMoment.isValid() || !endMoment.isValid()) {
+    return undefined
+  }
+  if (endMoment.isBefore(beginMoment)) {
+    endMoment = endMoment.add(1, 'day')
+  }
+
+  return endMoment.diff(beginMoment, 'minutes')
+}
+
```

Comment:
This function is not releated to maintance and windows. I calculates the differnence in minutes for two times in the 24h format.

Reply:
renamed to getDurationInMinutes